src/views/search/chrome/AppHeader.tsx

```diff
@@ -26,7 +26,7 @@ export const AppHeader = () => {
               <a
                 href={appHeader.logo.link ?? "https://vectara.com/"}
                 target="_blank"
-                className="appHeaderLogo"
+                className="appHeaderLogo" rel="noreferrer"
```

Comment:
I think `rel` should be defined on the next line but Prettier should be taking care of formatting stuff for us. We can figure out how to get it working properly in a separate scope.